### PR TITLE
cambricon: fix errors in merge code 0708 2

### PR DESCRIPTION
**Pull request by FuncSherl**
1、fix grid error and softmax error 
2、add autotune config for mm/bmm/mv
3、other log changes

### CODE REVIEW

File: benchmark/performance_utils.py [hunk 1]
```diff
@@ -43,7 +43,7 @@ def profile(self, op, *args):
         return latency
 
     def run(self):
-        print(f"Operator {self.op_name} Speedup Test ({self.dtype})")
+        # print(f"\nOperator {self.op_name} Speedup Test ({self.dtype})")
```
Comment (sethbrin) [inline benchmark/performance_utils.py:46]: why comment this line

File: src/flag_gems/ops/bmm.py [hunk 2]
```diff
@@ -6,6 +6,15 @@
 
 from ..utils import libentry
 
+# cambricon configs
+blocksmn = [256, 512, 1024]
```
Comment (sethbrin) [inline src/flag_gems/ops/bmm.py:10]: Could you please describe what this MR has done?

File: src/flag_gems/ops/softmax.py [hunk 3]
```diff
@@ -212,9 +212,28 @@ def softmax_kernel_inner(
 
 
 # ------------------------  backward -------------------------------
+@libentry()
+@triton.autotune(
+    configs=[
+        triton.Config({"TILE_K": 4}),
```
Comment (sethbrin) [inline src/flag_gems/ops/softmax.py:218]: for performance?

File: benchmark/performance_utils.py (hunk 1 above)
Comment (FuncSherl) [reply benchmark/performance_utils.py:46]: print log at last with format will help processing result

File: src/flag_gems/ops/softmax.py (hunk 3 above)
Comment (FuncSherl) [reply src/flag_gems/ops/softmax.py:218]: to avoid grid error and autotune as well

File: src/flag_gems/ops/bmm.py (hunk 2 above)
Comment (FuncSherl) [reply src/flag_gems/ops/bmm.py:10]: added describe

**Pull request merged**